core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_sidebar_directive.html

```diff
@@ -112,6 +112,9 @@
   simple-editor-sidebar .highlighted {
     background-color: #c5e1a5;
   }
+  simple-editor-sidebar .error {
```

Comment:
@andromfins  Adding the icon doesnt come in the class . We will have to add a new <i> tag which will show only when an error comes up . So should i do that in this issue or  a different one ?